src/main/java/com/synopsys/integration/jira/common/rest/service/PluginManagerService.java

```diff
@@ -77,16 +76,15 @@ public PluginManagerService(Gson gson, JiraApiClient jiraApiClient) {
         requestBuilder.method(HttpMethod.GET);
         requestBuilder.addHeader(ACCEPT_HEADER, MEDIA_TYPE_PLUGIN);
 
-        JiraResponse jiraResponse = jiraApiClient.get(requestBuilder.build());
-        if (jiraResponse.isStatusCodeError()) {
-            if (404 == jiraResponse.getStatusCode()) {
-                return Optional.empty();
+        try {
+            PluginResponseModel pluginComponent = jiraApiClient.get(requestBuilder.build(), PluginResponseModel.class);
```

Comment:
So the error handling has moved into the client code?

Reply:
I actually reverted back to the original here after changing it in my last PR. So the functionality will be the same as it previously was https://github.com/blackducksoftware/int-jira-common/commit/cd5ebf70bfddcc5166e3a7ac72b7f8ff92f098df#diff-0a4f2c5e551cd4fdcbb89df0c78ee3fd